src/cryojax/inference/transforms/_lie_group_transforms.py

```diff
@@ -0,0 +1,131 @@
+"""
+Implementation of transformations in local tangent spaces of lie groups.
+This is a re-implementation of `jaxlie.manifold`.
+
+This was necessary because `cryojax` has its own pattern for `jax`
+transformations, using `equinox`. The `jaxlie.manifold.grad` and
+`jaxlie.manifold.value_and_grad` are not compatible with the `cryojax`
+wrappers for gradient transformations.
```

Comment:
good to put these comments in

Reply:
You might want to point to specific parts in jaxlie.manifold. Or at least a specific version/commit of jaxlie.